libcontainer/specconv/example.go

```diff
@@ -155,7 +155,7 @@ func Example() *specs.Spec {
 	}
 }
 
-// ExampleRootless returns an example spec file that works with rootless
+// ToRootless returns an example spec file that works with rootless
 // containers. It's essentially a modified version of the specfile from
 // Example().
```

Comment:
Would you mind rewriting the comment entirely (since the description is actually wrong). Just use this (or similar):

```
// ToRootless converts the given spec file into one that should work with          
// rootless containers, by removing incompatible options and adding others that 
// are needed.
```

Reply:
Updated, PTAL. @cyphar 
Thanks for your kind feedback.